packages/by-name/boot-microvm/package.nix

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 Edgeless Systems GmbH
+# SPDX-License-Identifier: AGPL-3.0-only
+
+{
+  writeShellApplication,
+  qemu,
+  OVMF,
+}:
+
+# Usage example:
+# outPath=$(nix build .#kata.kata-image --print-out-paths); nix run .#boot-microvm -- "${outPath}/bzImage" "${outPath}/initrd" "${outPath}/image-podvm-gpu_1-rc1.raw" "$(nix eval --raw .#kata.kata-image.cmdline)"
```

Comment:
What do you think about taking the base `$outPath` instead of `$1 $2 $3` and figuring out the kernel, initrd and rootfs automatically? If the cmdline where in the outputs, too, invocation could be very simple...

Reply:
I wanted to have a simple utility that's not specific to this image. If we want to tie it to booting this image, i'd put it in it's `passthru`. That might actually be a good idea.